subprojects/platform-play/src/integTest/groovy/org/gradle/play/integtest/continuous/PlayMultiProjectReloadIntegrationTest.groovy

```diff
@@ -170,7 +170,7 @@ var message = "Hello JS";
         fails()
         !executedTasks.contains(':primary:runPlayBinary')
         errorPageHasTaskFailure(":submodule:compilePlayBinaryScala")
-        serverStartCount == 1
+        serverStarted()
```

Comment:
This also suggests we're missing a flush at the end of the build. We should guarantee that by the time that the build has completed, we've forwarded all of the log messages, regardless of whether we're in continuous mode or not.

Reply:
Yeah, this one was actually me just being a little too overzealous.  Id din't actually see this fail - it was the other change to this file (where the restart is not triggered by a build) where we need some polling.  I just changed this while I was in there, but its not necessary.

I've removed it.